spec/spec_with_project.rb

```diff
@@ -127,6 +127,16 @@ def refute_success_status(result)
       refute(result.status)
     end
 
+    sig { params(result: MockProject::ExecResult, snippet: String).void }
+    def assert_output_includes(result, snippet)
+      assert_includes(result.out, snippet, result.to_s)
```

Comment:
Does this still print the original failure message that includes the diff as well? If not, it would be great to have that since it is also helpful to know what was being expected for match.

Reply:
It does still print the original failure message 👍 

```
Expected "Requiring all gems to prepare for compiling...  Done\n\nRemoving RBI files of gems that have been removed:\n\n  Nothing to do.\n\n  Compiled tapioca\n      create  sorbet/rbi/gems/tapioca@0.9.2.rbi\n\nChecking generated RBI files...  Done\n\n  Changed strictness of sorbet/rbi/gems/tapioca@0.9.2.rbi to `typed: false` (conflicting with DSL files)\n\nAll operations performed in working directory.\nPlease review changes and commit them.\n" to include "Compiled tapioca1\n".```